server/internal/infrastructure/gcp/config_test.go

```diff
@@ -33,20 +37,21 @@ func TestCloudTasksConfig_buildQueueUrl(t *testing.T) {
 		},
 	}
 	for _, tt := range tests {
+		tt := tt
 		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
 			c := &CloudTasksConfig{
 				GCPProject: tt.fields.GCPProject,
 				GCPRegion:  tt.fields.GCPRegion,
 				QueueName:  tt.fields.QueueName,
 			}
 			got, err := c.buildQueueUrl()
-			if (err != nil) != tt.wantErr {
-				t.Errorf("CloudTasksConfig.buildQueueUrl() error = %v, wantErr %v", err, tt.wantErr)
+			if tt.wantErr {
+				assert.Error(t, err)
```

Comment:
Check errors more strictly

`wantErr bool` -> `wantErr error`

```suggestion
			if tt.wantErr != nil {
				assert.Equal(t, tt.wantErr, err)
```